contracts/mechs/nevermined/BalanceTrackerNvmSubscriptionNative.sol

```diff
@@ -37,14 +37,13 @@ contract BalanceTrackerNvmSubscriptionNative is BalanceTrackerFixedPriceNative {
     event SubscriptionSet(address indexed token, uint256 indexed tokenId);
     event RequesterCreditsRedeemed(address indexed account, uint256 amount);
 
-    // Credit to token ratio in 1e18 form
-    // N credits for M tokens, tokenCreditRatio = M * 10^18 / N
-    uint256 public immutable tokenCreditRatio;
-
     // Subscription NFT
     address public subscriptionNFT;
     // Subscription token Id
     uint256 public subscriptionTokenId;
+    // Credit to token ratio in 1e18 form
+    // N credits for M tokens, tokenCreditRatio = M * 10^18 / N
+    uint256 public tokenCreditRatio;
```

Comment:
Move `tokenCreditRatio` in the subscription setting. This allows not to rush with token-credit calculation during the NVM balance tracker deployment, and double check things at the time of the subscription setting.